src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import './Footer.scss';
+import GoogleIcon from '../../../assets/google.png';
+import MailIcon from '../../../assets/mail.png';
+import FacebookIcon from '../../../assets/facebook.png';
+import TwitterIcon from '../../../assets/twitter.png';
+
+const Footer = () => (
+  <footer className="footer">
+    <div className="contact-info">
```

Comment:
lets make our style names more semantic, because this is our contact info, we could rename to `ah-contact`

Reply:
Noted